src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -81,7 +81,7 @@
     public static final int DEPRECATED_LITERAL = JavadocParser.DEPRECATED_LITERAL;
 
     /**
-     * '@since' literal in @since Javadoc tag.
+     * '@since' literal in {@code @since} Javadoc tag.
```

Comment:
How come `'@since'` isn't flagged?

Reply:
due to `'`. It isn't a white space nor asterisk

Reply:
I assume `@since'` would be handled the same?
How does the javadoc tool handle a non-whitespace character after `@since`?

Reply:
> I assume `@since'` would be handled the same?
> How does the javadoc tool handle a non-whitespace character after `@since`?

it looks like the javadoc tool parser stops after first non-alphanumeric symbol:
`@Since1` => Error: unknown tag: Since1
`@since+3` => OK, renders as **since:** +3

I'll update the pattern

Reply:
The pattern should be simple `\s@\w+`

Reply:
> please also rename commit `fix violations` to something like `fix violations for JavadocBlockTagLocation`.

done